File: test/fixtures/node-got/post_with_quotes_anywhere.js
import got from 'got';

const response = await got.post('http://localhost:28139', {
  headers: {
    'A': "''a'",
    'B': '"',
    'Cookie': 'x=1\'; y=2"',
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  username: "ol'",
  password: 'asd"',
  body: 'a=b&c="&d=\''
});

File: test/fixtures/ruby-httparty/post_with_quotes_anywhere.rb
require 'httparty'

url = 'http://localhost:28139'
auth = { username: "ol'", password: 'asd"'}
headers = {
  'A': "''a'",
  'B': '"',
  'Cookie': 'x=1\'; y=2"',
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = 'a=b&c="&d=\''
res = HTTParty.post(url, basic_auth: auth, headers: headers, body: body)
